src/main/scala/ch/openolitor/stammdaten/models/AbosModels.scala

```diff
@@ -35,7 +35,7 @@ case class AboId(id: Long) extends BaseId
 object IAbo {
   def calculateAktiv(start: LocalDate, ende: Option[LocalDate]): Boolean = {
     val yesterday = LocalDate.now.minusDays(1)
-    start.isBefore(LocalDate.now) && (ende map (_.isAfter(yesterday)) getOrElse true)
```

Comment:
couldn't you replace the first part with:
!start.isAfter(now)

Reply:
done